src/main/resources/properties/secure/app-dev.yaml

```diff
@@ -17,8 +17,12 @@ splunk:
   token: "![yqKEWZ57M3XUasumfcwhV5A3JMblY6G4MCwHM5j5VuA=]"
   
 elk:
-  user: "user"
-  password: "password"
+  user: "![cClkzNzePN9/3L4Ag8XzsQ==]"
```

Comment:
Let's be careful with these values, seem to be related to a real ELK instance, let's keep the original ones